Sherlock/yoda/DOCU.md

```diff
@@ -355,11 +369,14 @@ api.Transaction([](PrimesAPI::T_DATA data) {
 // Primes that start with `4`.
 EXPECT_EQ("41,43,47",
           api.Transaction([](PrimesAPI::T_DATA data) {
-            // TODO(dkorolev): In real life, the order can be off. -- D.K.
             std::ostringstream os;
+            std::set<int> n;
             for (const auto cit : data[static_cast<FIRST_DIGIT>(4)]) {
-              os << ',' << (static_cast<int>(cit.row) * 10 +
-                            static_cast<int>(cit.col));
+              n.insert(static_cast<int>(cit.row) * 10 +
+                       static_cast<int>(cit.col));
+            }
+            for (int i : n) {
```

Comment:
Isn't there a templated container-friendly version? :( if no, let me add it
first thing I'm back from breakfast.

On Thursday, June 11, 2015, Maxim Zhurovich <notifications@github.com>
wrote:

> In Sherlock/yoda/DOCU.md
> <https://github.com/c5t/Current/pull/204#discussion_r32255340>:
>
> >              for (const auto cit : data[static_cast<FIRST_DIGIT>(4)]) {
> > -              os << ',' << (static_cast<int>(cit.row) * 10 +
> > -                            static_cast<int>(cit.col));
> > +              n.insert(static_cast<int>(cit.row) * 10 +
> > +                       static_cast<int>(cit.col));
> > +            }
> > +            for (int i : n) {
>
> Join works only with std::vector<std::string> :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/c5t/Current/pull/204/files#r32255340>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783